docs/semantics.md

```diff
@@ -228,6 +228,8 @@ However, implicit directories does have drawbacks:
 
 Alternatively, users can create a script which lists the buckets and creates the appropriate objects for the directories so that the ```--implicit-dirs``` flag is not used. 
 
+**Note:** `--implicit-dirs` flag is detrimental to performance, so it should be avoided unless necessary. To avoid it, you can mount using `--implicit-dirs` flag once, then create the directories on the mounted filesystem using the ```mkdir``` command, which will create the Cloud  Storage objects backing the directories, and then mount without --implicit-dirs  flag from the next time onwards.
```

Comment:
I am not sure if I want to make a strong statement like "**Note:** `--implicit-dirs` flag is detrimental to performance". There are simple ways to avoid the List calls
1. sufficient Metadata
2. script to convert.